src/component/mxgraph/BpmnMxGraph.ts

```diff
@@ -135,11 +134,10 @@ export class BpmnMxGraph extends mxgraph.mxGraph {
 
   private getZoomHandler(calculateFactorOnly: boolean) {
     return (event: Event, up: boolean) => {
```

Comment:
❓ To avoid to have `const evt = event as MouseEvent;`, is it not possible to have this ?
```suggestion
    return (event: MouseEvent, up: boolean) => {
```

Reply:
I have just checked. If I change the type, the calls to `debounce` and `throttle` generate a TSC error 😢 
```
TS2345: Argument of type 'DebouncedFunc<(event: MouseEvent, up: boolean) => void>' is not assignable to
parameter of type '(event: Event, up: boolean) => void'.
Types of parameters 'event' and 'event' are incompatible.
Type 'Event' is missing the following properties from type 'MouseEvent': altKey, button, buttons,
clientX, and 21 more.
```